README.md

```diff
@@ -1,7 +1,7 @@
 # LLaMA 
 
 This repository is intended as a minimal, hackable and readable example to load [LLaMA](https://ai.facebook.com/blog/large-language-model-llama-meta-ai/) ([arXiv](https://arxiv.org/abs/2302.13971v1)) models and run inference.
-In order to download the checkpoints and tokenizer, fill this [google form](https://forms.gle/jk851eBVbX1m5TAv5)
+In order to download the checkpoints and tokenizer, fill this [google form](https://forms.gle/jk851eBVbX1m5TAv5) or if you want to save our bandwidth use this BitTorrent link: "[magnet:?xt=urn:btih:ZXXDAUWYLRUXXBHUYEMS6Q5CE5WA3LVA&dn=LLaMA](magnet:?xt=urn:btih:ZXXDAUWYLRUXXBHUYEMS6Q5CE5WA3LVA&dn=LLaMA)".
 
 ### Setup
```

Comment:
buenas como hago para conseguir llama version 65b.  o el torrent. porque estoy rastreando hace rato para instalar y probar encontre los pesos peor me faltan archivos de instalacion, si alguno me puede alyudar le agradezco , aunque sea por mje.privado. slot1_@hotmail.com.

Reply:
@ADANCE-STACK were you able to download it? I'm also looking for it

Reply:
Downloading the Llama 7B, taking forever...